dias/analyzers/feedpositions_analyzer.py

```diff
@@ -68,6 +73,14 @@ class FeedpositionsAnalyzer(CHIMEAnalyzer):
     Labels
         source : Source transit name.
 
+    dias_task_<task name>_bad_feeds_percent
```

Comment:
```suggestion
    dias_data_<task name>_bad_feeds_percent
```

Reply:
task metrics get the prefix `dias_task_` while data metrics get `dias_data_`.

Reply:
okay I will also correct for line 68 then where you also used task

Reply:
Good catch! Thanks